plugin/hosts/hosts_test.go.orig

```diff
@@ -0,0 +1,79 @@
+package hosts
```

Comment:
Should this file be included?

Reply:
Thank you, no, it should not and was removed.